main.py

```diff
@@ -156,8 +156,8 @@ async def nini(ctx):
         c += 1
         auth = m.author.name
         if auth not in all_losers:
-            all_losers[auth] = [0, 0, 0, 0]
-        all_losers[auth][NINI_MESSAGES] += 1
+            all_losers[auth] = dict()
+        all_losers[auth]["NINI_MESSAGES"] += 1
```

Comment:
It may be best to change "NINI_MESSAGES" to something prettier like "msgs" or "messages", to keep only the global variables as full uppercase names.

Same for every other items.

Reply:
Other than that, LGTM. I will merge after this change.